src/common/utils/disposer.ts

```diff
@@ -3,23 +3,35 @@
  * Licensed under MIT License. See LICENSE in root directory for more information.
  */
 
-export type Disposer = () => void;
 
-interface Extendable<T> {
-  push(...vals: T[]): void;
+
```

Comment:
Are changes in this file part of the fix?

Reply:
Yes, because the `this.xterm.onResize` returns a `Disposable`